wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -29,7 +29,7 @@ <h1 class="w-header__title" id="header-title">
                     {% elif avatar %}
                         <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
                     {% endif %}
-                    {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
+                    {{ title }}{% if subtitle %}<span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
```

Comment:
this was correct. you do want a space between the title and the subtitle

Reply:
🤔
Yes

Reply:
Indeed yes. These spacing things with Django tags always confuse me. Thanks @zerolab!